.github/DISCUSSION_TEMPLATE/surveys.yaml

```diff
@@ -0,0 +1,92 @@
+title: "User Feedback Survey: [<Community Name>]"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        We value your feedback and would love to hear about your experience with our current web application.
+        Your insights will be instrumental in shaping the new diracx-web client we are building.
+        Please take a moment to answer the questions below.
+
+  - type: textarea
+    id: question-1
+    attributes:
+      label: "1. Describe your overall experience with the current [DIRAC web application](https://github.com/DiracGrid/webAppDIRAC)."
+      description: >
+        Focus on aspects like ease of use, reliability, and speed
+    validations:
+      required: true
+
+  - type: textarea
+    id: question-2
+    attributes:
+      label: "2. What tasks do you primarily use our web application for?"
+      description: >
+        Example: Users monitor their jobs.
+    validations:
+      required: true
+
+  - type: textarea
+    id: question-3
+    attributes:
+      label: "3. What features do you find most useful and why?"
+      description: >
+        Example: The possibility of monitoring my jobs in real-time.
+    validations:
+      required: true
+
+  - type: textarea
+    id: question-4
+    attributes:
+      label: "4. Are there any features or functionalities you find frustrating or difficult to use?"
+      description: >
+        Example: Logs are hard to read.
+    validations:
+      required: true
+
```

Comment:
Maybe it's worth separating user needs from admin needs for some of these questions? Espeically things which cause confusion from users.